src/components/ReplyForm/types.ts

```diff
@@ -0,0 +1,21 @@
+import { EditorState } from 'draft-js';
+
+export type PropsFromState = {
+    cursorPosition: number;
+};
+
+export type ContainerProps = {
```

Comment:
Seems like these should be defined and exported from the component and imported by the container. The "container props" is really just `cursorPosition`, right?